packages/notification-services-controller/src/NotificationServicesPushController/services/push/push-web.ts

```diff
@@ -12,7 +12,7 @@ import type { Types } from '../../../NotificationServicesController';
 import { Processors } from '../../../NotificationServicesController';
 import type { PushNotificationEnv } from '../../types/firebase';
 
-const sw = self as unknown as ServiceWorkerGlobalScope;
+declare const self: ServiceWorkerGlobalScope;
```

Comment:
self does not exist on mobile and breaks. So instead removing the `sw` variable and `self` is only used within functions instead of a global variable.

In the future (once we support multiple exports and tree shaking) we can tidy this up.